regolith/builders/grantreportbuilder.py

```diff
@@ -23,6 +23,7 @@
 
 
 def subparser(subpi):
+    subpi.add_argument("authorFullName", help="full name of the author of this grant report")
```

Comment:
this is very brittle.  I doubt I will ever remember the full name of people and whether it is capitalized and spaces and whatnot.  We have two choices here.
1. require the person id (e.g., you would be vlin).  This is easier to remember because we use a mostly standard naming scheme.
1. allow anything that is in id, name or alias and then use fuzzy retrieval to get the person back.

(2) is probably better because (1) is also covered by (2)